gallery/src/components/NxBinaryDonutChart/NxBinaryDonutChartEmptyExample.tsx

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { NxBinaryDonutChart } from '@sonatype/react-shared-components';
+
+export default function NxBinaryDonutChartEmptyExample() {
+  return <NxBinaryDonutChart percent={0} />;
```

Comment:
This could just be merged into the "minimal" example

Reply:
> This could just be merged into the "minimal" example

Done 76104c42b68ca9b5b18665b3149a349594ed02e3